tests/src/system/basic/CLIRuleTests.java

```diff
@@ -81,9 +81,37 @@ public void rule1to1() throws Exception {
             String activationId = activationIds.get(0);
             assertTrue("Expected message not found: " + expected, wsk.logsForActivationContain(activationId, expected, DELAY));
         } finally {
-            wsk.delete(Action, "A_121");
+        	wsk.delete(Rule, "R_121");
+        	wsk.delete(Action, "A_121");
             wsk.delete(Trigger, "T_121");
-            wsk.delete(Rule, "R_121");
+        }
+    }
+
+    /**
+     * rule test with one trigger and one action (action has space in name)
+     */
+    @Test
+    public void rule1to1WithSpaceInActionName() throws Exception {
```

Comment:
Sorry, but what exactly do you mean?
Is your intention to test `wsk rule create r1 t1 /whisk.system/samples/wc`?
Isn't this another issue, or am I wrong?